### DISCUSSION TITLE
Mod selection shortkeys cannot be configured

### DISCUSSION THREAD

**Discussion by hashelq** [ppy/osu#18453] [Ideas] [answered]
Some people may prefer to change their keyboard layouts on board, for example: people who have programmable keyboards that support QMK firmware.
By changing the keyboard in the keyboard, user remaps keys to allow him to use his keyboards with preferred layout at any device he wants. Imagine that some keyboard is programmed to send `g` when pressing `w` because someone has programmed his keyboard to do so.
As i can see there is no options in key configuration panel to change the mod shortcuts. By adding this it will become easier to navigate in the menu and will allow people who have custom kb layouts not to suffer from trying to figure out where the specific shortcut is placed.

**Comment by bdach** [comment, marked as answer]
this is partially covered by https://github.com/ppy/osu/issues/11143. i am unsure whether we will permit customisability of these keybinds. i'm leaning "no" to avoid unnecessary complexity.
